pkg/network/networker.go

```diff
@@ -240,7 +243,10 @@ func (n *networker) Leave(networkdID pkg.NetID, containerID string) error {
 
 // ZDBPrepare sends a macvlan interface into the
 // network namespace of a ZDB container
-func (n networker) ZDBPrepare(hw net.HardwareAddr) (string, error) {
+func (n networker) ZDBPrepare(id string) (string, error) {
+
+	hw := ifaceutil.HardwareAddrFromInputBytes([]byte("pub:" + id))
```

Comment:
I think the "pub:" prefix should not be here, so the input is the same as it was previously for this iface. Otherwise, if a node has a zdb running and reboots after this update, it will generate a new mac address, resulting in a different ip6 from slaac, which will mean the ipv6 originally returned in the result (which is probably used) is not valid anymore